docs/guide/06-Running-Application.md

```diff
@@ -79,29 +79,28 @@ import Prelude
 import Data.Maybe (Maybe(..))
 import Effect (Effect)
 import Effect.Console (log)
-import Example.Driver.IO.Button as B
-import FRP.Event as Event
 import Halogen (liftEffect)
 import Halogen as H
 import Halogen.HTML as HH
 import Halogen.Aff as HA
 import Halogen.HTML.Events as HE
 import Halogen.HTML.Properties as HP
+import Halogen.Subscription as SUB
```

Comment:
Do you mind switching this to:

```suggestion
import Halogen.Subscription as HS
```

That keeps it the same as everywhere else we import this. Thanks!